src/env.ts

```diff
@@ -13,7 +13,8 @@ function loadFromFile(env = 'test') {
   return parsed!; // if error is undefined, then parse is defined
 }
 
-export default {
+// add-modules-exports doesn't work with ts!?
+exports.default = {
```

Comment:
is there a point to doing this if we're only consuming the CJS transpiled version?

Reply:
probably not!